src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
-export interface Figure {
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Colors = 'red' | 'green' | 'blue';
```

Comment:
```suggestion
type ShapesType = 'triangle' | 'circle' | 'rectangle';
type ColorsType = 'red' | 'green' | 'blue';
```

Reply:
can help you understand what shapes and colors are

Reply:
It's the same as to name constants like `${name}Const`, overkill
From TS syntax it's clear that it's a type